src/parser.ts

```diff
@@ -163,6 +163,22 @@ function findBinaryFunction(tokens: ExprElement[], fn: string) {
   }
 }
 
+// Some minuses have been parsed as functions (i.e. an `ExprFunction` with a single argument).
+// If they are preceded by something, we need to treat the expression as a unary minus.
```

Comment:
Hello, what do you think about rephrasing this comment to something like this:
```
// Some minuses are initially parsed as unary functions (i.e. the string "- b" is parsed an `ExprFunction` with
// `.fn` = "−" and an `ExprIdentifier` "b" as its single argument.)
// If any unary minus function is preceded by another token, we need to merge the pair of tokens into a binary minus
// function.
```

Reply:
Definitely clearer!